cache/cache.go

```diff
@@ -241,7 +241,11 @@ func (c *Cache) get(ctx context.Context) (*clientv3.GetResponse, error) {
 func (c *Cache) watch(rev int64) error {
 	readyOnce := sync.Once{}
 	for {
-		storeW := newWatcher(c.cfg.PerWatcherBufferSize, nil)
+		buf := c.cfg.PerWatcherBufferSize
+		if buf < 1 {
+			buf = 1
```

Comment:
Like In previous case, please don't override user configuration with some minimal values. Reject invalid configuration, in the cache setup.